aim/web/ui_v2/src/services/api/metrics/metricsService.ts

```diff
@@ -12,7 +12,7 @@ function getMetricsData() {
     call: () => ({
       then: (resolve: (data: IRun[]) => void, reject?: unknown) => {
         setTimeout(() => {
-          const mock = generateMetrics(500, 200);
+          const mock = generateMetrics(100, 50);
```

Comment:
It seems interactions slow down when the number of visualized lines becomes greater than 200.

Reply:
Yeah, indeed, but this is actually not related to this PR only.
We've noticed the same issue for hover attributes rendering on screens with high resolution.
We will make an investigation and try to resolve this issue within a separate PR.

Reply:
Got it, great.